pkgs/by-name/ta/taler-depolymerization/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchgit,
+  stdenv,
+  darwin,
+}:
+rustPlatform.buildRustPackage {
+  pname = "taler-depolymerization";
+  version = "0-unstable-2024-06-17";
+
+  src = fetchgit {
+    url = "https://git.taler.net/depolymerization.git/";
+    rev = "a0d27ac3bba22d4934ca9f7b244b0d9e45bb484f";
+    hash = "sha256-HmQ/DPq/O6aODWms/bSsCVgBF7z246xxfYxiHrAkgYw=";
+  };
+
+  cargoHash = "sha256-NgoLCTHhEs45cnx21bU2ko3oWxePSzKgUpnCGqhjvTs=";
+
+  outputs = [
+    "out"
+    "doc"
+  ];
+
+  postPatch = ''
+    # fix missing expression in test docs
+    substituteInPlace common/src/status.rs \
+      --replace-fail '          -> Requested' '()        -> Requested'
+  '';
+
+  postInstall = ''
+    mkdir -p $doc/share/doc $out/share/examples
+
+    cp -R docs $doc/share/doc/taler-depolymerization
+    cp docs/*.conf $out/share/examples
+  '';
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.CoreFoundation
+    darwin.apple_sdk.frameworks.Security
+    darwin.apple_sdk.frameworks.SystemConfiguration
+  ];
+
+  meta = {
+    description = "Wire gateway for Bitcoin/Ethereum";
+    homepage = "https://git.taler.net/depolymerization.git/";
+    license = lib.licenses.agpl3Only;
+    maintainers = [
+      # maintained by the team working on NGI-supported software, no group for this yet
```

Comment:
Maybe would make sense to create it, if you plan on adding yourself in it?

Reply:
I don't think this is a decision I should make on my own. What do you think @fricklerhandwerk?

Reply:
Having a team would be good thing to distinguish NGI0 packages and find them programmatically. We don't *really* have the capacity to maintain all of them, but we could try. At worst, the team could be empty simply as a beacon, and we maintain what we can best-effort.

Reply:
Yes, that works for me!

Reply:
I would also like to join the team, if that's okay.